cdap-app-fabric/src/main/java/co/cask/cdap/metadata/MetadataHttpHandler.java

```diff
@@ -76,7 +77,8 @@ public MetadataHttpHandler(MetadataAdmin metadataAdmin) {
   public void getAppMetadata(HttpRequest request, HttpResponder responder,
```

Comment:
I'll add support for a 'scope' query param in a later PR. This PR is parity with current functionality + ability to create a system metadata dataset only.